web/src/js/__tests__/ducks/tutils.js

```diff
@@ -8,4 +14,58 @@ export function createStore(parts) {
     )
 }
 
-export { default as TFlow } from './_tflow'
+export { TFlow }
+
+export function TStore(){
+    let tflow = new TFlow()
+    return mockStore({
+        ui: {
```

Comment:
If we use the reducers, it will look like this:
```javascript
export function TStore(){
    let tflow = new TFlow(),
        uiState = null,
        setttingState = null
    uiState = reduceUI(undefined, UIActions.setContentViewDescription('foo'))
    uiState = reduceUI(uiState, UIActions.setContent(['foo', 'bar']))
    uiState = reduceUI(uiState, UIActions.startEdit(tflow))
    setttingState = reduceSetting(setttingState, {type: SettingActions.UPDATE, data: { contentViews: ['Auto', 'Raw', 'Text']}})
    setttingState = reduceSetting(setttingState, {type: SettingActions.UPDATE, data: { anticache: true }})
    setttingState = reduceSetting(setttingState, {type: SettingActions.UPDATE, data: { anticomp: false }})
    return mockStore({
        ui: uiState,
        settings: settingState
        })
}
```
Basically, we need to pass a bunch of actions to the reducers to get the state object, it's a little bit redundant. Would it be more straightforward and clear to define the state object statically? In that way we could see the structure of the store object and modify some value easily.